internal/zinc-apiinfo/src/main/scala/xsbt/api/HashAPI.scala

```diff
@@ -249,7 +249,7 @@ final class HashAPI private (
     hashTypeParameters(c.typeParameters)
     hashType(c.selfType)
     if (includeSealedChildren)
-      hashTypes(c.childrenOfSealedClass, includeDefinitions)
+      hashTypesSymmetric(c.childrenOfSealedClass, includeDefinitions)
```

Comment:
Could we add test for this?

Reply:
Sure, I can try.  Any tips where/how?

Reply:
Some potential places to get hints from:

- https://github.com/sbt/zinc/blob/develop/internal/zinc-apiinfo/src/test/scala/xsbt/api/HashAPISpecification.scala
- https://github.com/sbt/zinc/blob/develop/internal/zinc-apiinfo/src/test/scala/sbt/internal/inc/ClassToAPISpecification.scala
- https://github.com/sbt/zinc/blob/develop/zinc/src/sbt-test/apiinfo/unstable-existential-names/test